SmithyDafnyMakefile.mk

```diff
@@ -82,47 +82,45 @@ ENABLE_EXTERN_PROCESSING?=
 # Verify the entire project
 verify:Z3_PROCESSES=$(shell echo $$(( $(CORES) >= 3 ? 2 : 1 )))
 verify:DAFNY_PROCESSES=$(shell echo $$(( ($(CORES) - 1 ) / ($(CORES) >= 3 ? 2 : 1))))
+verify:DAFNY_OPTIONS=--allow-warnings
```

Comment:
Curious, what's the reasoning to take this out of the commands and put it as DAFNY_OPTIONS? It looks like you added it back everywhere?

Reply:
ahh i forgot to add a meta comment about this.
--allow-warnings shouldn't be part of the overall dafny transpile/verify commands since we want to remove it sometime soon. Making it an option even though it is still used everywhere made sense to me